node/node.go

```diff
@@ -431,6 +424,11 @@ func (n *Node) createMetaChainBootstrapper(rounder consensus.Rounder) (process.B
 		return nil, err
 	}
 
+	err = bootstrap.SetStatusHandler(n.GetAppStatusHandler())
```

Comment:
Please remove this setStatusHandler. It is already done on the interface.

Reply:
done